CONTRIBUTING.md

```diff
@@ -22,6 +22,44 @@ Runs tests and checkstyle.
 ./gradlew check
 ```
 
+## Installing/testing against a local maven repository
+
+Sometimes its helpful to build and install the bugsnag-java libraries into a
+local repository and test the entire dependency flow inside of a sample
+application.
+
+To get started:
+
+1. In the `bugsnag-java` directory, run
+   `./gradlew -Preleasing=true publishToMavenLocal`.
+   This installs `bugsnag-java` and `bugsnag-spring` into your local
+   maven repository.
+2. In your sample application `build.gradle`, add `mavenLocal()` to the *top* of
```

Comment:
Do we also need to supply instructions for Maven builds?

Reply:
As this is mainly intended for internal use, I think we can skip that for now